compiler_opt/es/regalloc_trace/regalloc_trace_worker.py

```diff
@@ -53,9 +60,65 @@ def _setup_base_policy(self):
     saver.save(self._tf_base_temp_dir)
     self._tf_base_policy_path = os.path.join(self._tf_base_temp_dir, "policy")
 
-  def __init__(self, *, gin_config: str, clang_path: str,
-               basic_block_trace_model_path: str, thread_count: int,
-               corpus_path: str):
+  def _maybe_copy_corpus(self, corpus_path: str,
```

Comment:
should this be a corpus.py utility?

Reply:
also, in this scenario (trace-based etc...) do we always want to copy the whole corpus, or just specific modules?

Reply:
> should this be a corpus.py utility?

I think it's a bit too specific to be generic to `corpus.py`, and no one else needs this functionality currently. If other workers in the future end up needing to copy the corpus locally, we can potentially look at generalizing it and moving it to somewhere like `corpus.py`.

> also, in this scenario (trace-based etc...) do we always want to copy the whole corpus, or just specific modules?

We want to copy the whole corpus, at least for now. The corpus that is being copied would have already been subsetted/minimized.